### PR TITLE
Allow github reviews to add/remove LGTM

### PR DESCRIPTION
**Pull request by fejta**
/hold

What channels should we go through for this sort of proposal?
/cc @howardjohn @cjwagner @utka 

ref https://github.com/istio/test-infra/issues/1433

NOTE: you can still `/lgtm` or `/approve` whenever you want to just lgtm or approve, respectively (aka current behavior remains the same).

These plugins did not behave this way initially because they predate github's review functionality.

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]
/lgtm

The best channel is probably just asking on #test-and-release or more formally at the test and release WG meeting (every tuesday). 

I don't know that I have ever actually seen someone `/approve` without an lgtm though so I don't think there would be any concerns with it. I have seen people give gituhb approval without realizing it did nothing many times though :slightly_smiling_face:

**Comment by howardjohn** [comment]
In test and release meeting we decided we want this, feel free to cancel

**Comment by fejta** [comment]
Great!

/hold cancel

**Pull request merged**
